en/docs/references/pricing.md

```diff
@@ -0,0 +1,227 @@
+# Pricing
+
+## Introduction
+
+Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can use Choreo by paying only for your usage: nothing more, nothing less. Choreo has payment plans for all types of users. You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you use more than 5,000 [steps](#what-is-a-step) per month. If you plan to use Choreo significantly, you can also opt for an Enterprise plan and enjoy special discount rates.
+
+## Plans
+
+|               | Free                   |  Pay As You Go    | Enterprise     |
+|---------------|------------------------|-------------------|----------------|
+| Cost Per Step | 5,000 steps per month *|$5 per 1,000 steps <br/> <ul><li>First 5,000 steps per month are free</li></ul>| Contact Us For Discounted Pricing <br/><ul><li> Volume Discounts</li><li> Committed-use Discounts</li><li> Support Channels and Support SLA</li></ul>|
+
+ \* Limited to a maximum of 3 components
```

Comment:
What is limited to a maximum of three components?

If the number is less than 10, we should use the word (in this case "three") instead of the integer.

Reply:
Actually according to https://developers.google.com/style/numbers#more-examples, if it's a unit, then recommended way is to use the number.

Reply:
This guide you have linked states the following:

"Numbers as words
In general, spell out the following:

Numbers from zero through nine, except as noted in [Numbers as numerals](https://developers.google.com/style/numbers#numbers-as-numerals)."

Reply:
If you look at the more recommendations you'll see similar usage. It doesn't make it readable by saying three components.

Reply:
In this case, this is neither a number without a unit and nor is it used in a mathematical expression. Therefore, it seems more appropriate to use the first guideline. The list of examples is also more in line with that.

Reply:
As discussed offline, since it is consistent across the doc, and as the guideline is a bit blurred. let's leave this as it is.